### PR TITLE
feat(deps): update generator deps for react-intl and parrot-generator

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

File: packages/generator-one-app-module/__tests__/app.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ describe('generator-one-app-module', () => {
           '@americanexpress/one-app-ducks': '^4.0.0',
           immutable: '^3.8.2',
           'prop-types': '^15.5.9',
-          'react-intl': '^3.6.0',
+          'react-intl': '^4.5.3',
```
Comment (infoxicator) [inline packages/generator-one-app-module/__tests__/app.js:122]: This is a major version update, did you test it on a `react-intl` enabled module? also worth a try on the `cultured-frankie` and see if the integration tests still pass

Comment (JamesSingleton) [reply packages/generator-one-app-module/__tests__/app.js:122]: I installed the latest version of `react-intl` to cultured frankie and run the integration tests and everything passes. Also verified this with a local module as well.

Comment (mtomcal) [reply packages/generator-one-app-module/__tests__/app.js:122]: So will we have an integration test PR in One App for this upgrade?

Comment (infoxicator) [reply packages/generator-one-app-module/__tests__/app.js:122]: at some point I think, a lot of the frankies need updating anyway

**Pull request merged**